app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    words = command.split(" ")
```

Comment:
no need to use space inside split method, it uses space by default
also u can unpack this like this, to avoid using indexes and increase a readability 
`arg1, arg2, arg3 = command.split()`